integration/framework.go

```diff
@@ -350,17 +350,19 @@ func (f *framework) initGuestClientset() error {
 
 func (f *framework) WaitForGuestReady() error {
 	if err := f.initGuestClientset(); err != nil {
-		return microerror.Maskf(err, "unexpected error initializing guest clientset")
+		return microerror.Maskf(err, "initializing guest clientset")
 	}
 
 	if err := f.waitForAPIUp(); err != nil {
-		return microerror.Mask(err)
+		return microerror.Maskf(err, "waiting for API being up")
 	}
 
 	if err := f.WaitForNodesUp(minimumNodesReady); err != nil {
-		return microerror.Mask(err)
+		return microerror.Maskf(err, "waiting for nodes being up")
```

Comment:
We discussed we should avoid adding annotations to returned errors. Is there something special about this case?